requirements.txt

```diff
@@ -19,3 +19,5 @@ lxml
 eviltransform
 stravaweblib
 tenacity
+numpy
+tzlocal
```

Comment:
we not not need this, you can use other function in this repo, that can get local time

Reply:
no i use it to get ueser's timezone. or we have a function to get this in this repo?